lib/foreman/renderer/errors.rb

```diff
@@ -0,0 +1,35 @@
+module Foreman
+  module Renderer
+    module Errors
+      class RenderingError < Foreman::Exception
+        def initialize(**params)
+          super(N_(self.class::MESSAGE), params)
```

Comment:
The MESSAGE constant must be passed in normally actually:

    super(self.class::MESSAGE, params)